docs/developer/semantic-version.rst

```diff
@@ -12,13 +12,12 @@ The formal convention of the version releases for specifying compatibility in NE
 It is incremented for the compatibility with the NEST Simulator.
 It implies that the major version of the NEST Desktop (2.x.x) has to match with the one of NEST Simulator (currently 2.x.x).
 
-
 **A minor number**
 
 It is a breaking feature such as a new library or a minor changes of the data structure.
 It means that this version could cause the compatibility issues and the user might reset the data of the page.
 
 **A patch number**
 
-It is a bugfix and non-breaking features were added to the code.
+It is a bug-fix and non-breaking features were added to the code.
```

Comment:
Where does this spelling come from? It seems to be a quite specific term (not contained in OLD,  MW, CD or PONS). I found the single word spelling in most cases, c.f. [Wiktionary](https://en.wiktionary.org/wiki/bugfix).
```suggestion
It is a bugfix and non-breaking features were added to the code.
```